src/components/ListItem/ListItem.js

```diff
@@ -32,8 +32,12 @@ const RenderDataItem = ({
   const textToDisplay = !shouldDisplayAnswer
     ? text
     : showAsCorrect
-    ? 'Correct'
-    : 'Wrong';
+    ? lang === 'en'
+      ? 'Correct'
+      : 'Marina'
+    : lang === 'en'
+    ? 'Wrong'
+    : 'Diso';
```

Comment:
Please create a new file with translation object that will be used in whole app (\translations\index.js):
```


// string ids
export const ANSWER_CORRECT = 'ANSWER_CORRECT';
export const ANSWER_WRONG = 'ANSWER_WRONG';

export const LANG_DATA = {
  [ANSWER_CORRECT]: {
    [LANGUAGE_NAMES.EN]: 'Correct',
    [LANGUAGE_NAMES.MG]: 'Marina',
  },
  [ANSWER_WRONG]: {
    [LANGUAGE_NAMES.EN]: 'Wrong',
    [LANGUAGE_NAMES.MG]: 'Diso',
  },
};

```
then use it like this :
```

 const answerCorrectText = LANG_DATA[ANSWER_CORRECT][lang];
  const answerWrongText = LANG_DATA[ANSWER_WRONG][lang];

  const textToDisplay = !shouldDisplayAnswer
    ? text
    : showAsCorrect
    ? answerCorrectText
    : answerWrongText;
```

Reply:
follow the pattern for any app string that need to be translated

Reply:
I like this approach a lot!

Reply:
So do I.